concordium-consensus/src/Concordium/Scheduler/ProtocolLevelTokens/Module.hs

```diff
@@ -397,16 +396,23 @@ queryTokenModuleState = do
     tmsMetadata <-
         getTokenState "metadata" >>= \case
             Nothing -> pltError $ QTEInvariantViolation "Missing 'metadata'"
-            Just metadata -> either corruptMetadataError return $ tokenMetadataUrlFromBytes $ LBS.fromStrict metadata
+            Just metadata ->
+                either (corruptDataError "metadata url") return $ tokenMetadataUrlFromBytes $ LBS.fromStrict metadata
+    tmsGovernanceAccount <-
+        getTokenState "governanceAccount" >>= \case
+            Nothing -> pltError $ QTEInvariantViolation "Missing governance account"
+            Just govAccount -> do
+                either (corruptDataError "governance account address") return $
+                    decodeFromBytes decodeCborTokenHolder "cbor token holder" (LBS.fromStrict govAccount)
```

Comment:
This has been changed so that what is stored is the index, not the address. We need to get the address. I would suggest adding `getAccountByIndex` to `PLTKernelQuery` and then using `getAccountCanonicalAddress`.